### DISCUSSION TITLE
Password managers not compatible

### DISCUSSION THREAD

**Discussion by timdams** [SafeExamBrowser/seb-win-refactoring#407] [Q&A] [answered]
Hello,
Our students have to login into our learning platform (moodle) on which they will perform their tests in SEB. However, most of them nowadays use a password manager as they have long and secure passwords for all their sites. Currently, student are forced to resort to a very unsafe method in order to use SEB and still be able to login: i.e. write the password down on paper.

Do you have any tips on how we could solve this?

**Comment by dbuechel** [comment, marked as answer]
If the password manager is a standalone application, then you could simply add it to the SEB configuration as an allowed third-party application (see https://safeexambrowser.org/windows/win_usermanual_en.html#ApplicationsPane). If it is a browser plugin or alike, then there is most likely nothing we can do, as SEB doesn't support browser plugins at the moment.
